Moonlight macOS/ViewController/StreamFrameViewController.m

```diff
@@ -0,0 +1,123 @@
+//
+//  StreamFrameViewController.m
+//  Moonlight macOS
+//
+//  Created by Felix Kratz on 09.03.18.
+//  Copyright © 2018 Felix Kratz. All rights reserved.
+//
+
+#import "StreamFrameViewController.h"
+#import "VideoDecoderRenderer.h"
+#import "StreamManager.h"
+#import "Control.h"
+#import "Gamepad.h"
+#import "keepAlive.h"
+
+@interface StreamFrameViewController ()
+@end
+
+@implementation StreamFrameViewController {
+    StreamManager *_streamMan;
+    StreamConfiguration *_streamConfig;
+    NSTimer* _timer;
+    ViewController* _origin;
+    struct Gamepad_device* device;
+}
+
+-(ViewController*) _origin {
+    return _origin;
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [keepAlive keepSystemAlive];
+    self.streamConfig = _streamConfig;
+    Gamepad_detectDevices();
```

Comment:
It looks like `Gamepad_detectDevices()` should only be called after `Gamepad_init()`

Reply:
It seems like Gamepad_detectDevices() is not required at all for initializing the gamepad.
Even when connecting during streaming there is no problem at all. I will look into this when implementing multi controller support.